testdata/tls_client-cert.pem

```diff
@@ -1,14 +1,14 @@
 -----BEGIN CERTIFICATE-----
-MIICOzCCAeGgAwIBAgIUEIohRQr9GVEsV3PzFHI7GKNvhxIwCgYIKoZIzj0EAwIw
+MIICOzCCAeGgAwIBAgIURmLWzhpEjx+vQx4JbV5B8C4DRPcwCgYIKoZIzj0EAwIw
```

Comment:
I think you'll need to rebase on top of the other PR I just merged that updated the four certs.